nilearn/datasets/__init__.py

```diff
@@ -0,0 +1,23 @@
+from .struct import (fetch_icbm152_2009, load_mni152_template, fetch_oasis_vbm)
+from .func import (fetch_haxby_simple, fetch_haxby, fetch_nyu_rest,
+                   fetch_adhd, fetch_miyawaki2008,
+                   fetch_localizer_contrasts, fetch_abide_pcp,
+                   fetch_localizer_calculation_task)
+from .atlas import (fetch_atlas_craddock_2012, fetch_craddock_2012_atlas,
+                    fetch_atlas_destrieux_2009, fetch_atlas_harvard_oxford,
+                    fetch_harvard_oxford, fetch_atlas_msdl, fetch_msdl_atlas,
+                    fetch_atlas_power_2011, fetch_atlas_smith_2009,
+                    fetch_smith_2009, fetch_atlas_yeo_2011,
+                    fetch_yeo_2011_atlas)
+from .utils import _fetch_files
```

Comment:
I don't think that we want _fetch_files in __init__. It's a private stuff for our own use.